src/modules/partition/Config.cpp

```diff
@@ -46,6 +46,32 @@ Config::swapChoiceNames()
     return names;
 }
 
+const NamedEnumTable< Config::LuksGeneration >&
+Config::luksGenerationNames()
+{
+    static const NamedEnumTable< LuksGeneration > names { { QStringLiteral( "luks1" ), LuksGeneration::Luks1 },
+                                                          { QStringLiteral( "luks2" ), LuksGeneration::Luks2 } };
```

Comment:
You can use `// clang-format: off` if you want to write this in a less far-indented fashion, so it's readable and clang-format doesn't mess it up. You can also write aliases here, e.g. add *luks* as an alias for *luks1*. And if you list it first, it's the "default" for some operations.

Reply:
The other `NamedEnumTable`s of this file are formatted in this way (without formatting off) I opted for the same here for consistency within the file.